README.md

```diff
@@ -19,37 +19,44 @@ To be able to build the docs locally, you need to install the following:
 
 You can build the docs for local development using the following command:
 
-```sh
-make serve-dev-draft
+```console
```

Comment:
[q] Why change all of these?

Reply:
Think of it kind of like our `command-line` styling in the docs, where we include a prompt character ($) to indicate an interactive snippet, but a full shell script (even a one-liner) doesn't include the prompt. It's a subtle piece of UX, and currently the GitHub markdown preview doesn't do anything special with it, but technically it could someday. And in the meantime, it's very very very slightly better for folks with screen readers, so I try to follow the convention when possible (and spread the word, now you've hopefully been infected by the `console` mind-virus).